tools/project_manager/external_catalog_search.php

```diff
@@ -169,11 +172,15 @@ function do_search_and_show_hits()
             continue;
         }
 
-        //if it's not a book don't display it.  we might want to uncomment in the future if there are too many records being returned - if (substr(yaz_record($id, $start, "raw"), 6, 1) != "a") { $start++; continue; }
         /** @phpstan-ignore-next-line */
         $marc_record = new MARCRecord();
         $marc_record->load_yaz_array($rec);
-
+        // If it's not a 'language material' (printed or typed) or
+        // 'manuscript language material' (written), don't display it.
+        // if (!in_array($marc_record->get_type_of_record(), ["Language material", "Manuscript language material"])) {
```

Comment:
Why the commented code?

Reply:
I was initially trying to figure out how to express 'language material only' in the query to the LoC server which would avoid the need to filter the response,  but I was getting weird and inconsistent results.

The original was commented, and I was undecided if I wanted to enable it or not.

Then I thought I might use this logic to decorate each table entry with an HTML class so we can selectively show/hide them in the browser with some javascript.

So... I'm undecided what I want to do with this, basically.